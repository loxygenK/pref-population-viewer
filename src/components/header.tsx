import styles from "./header.module.scss";

export const Header: React.FC = () => {
  return (
    <header className={styles.header}>
      <h1 className={styles.title}>pref-population-viewer</h1>
    </header>
  );
};
