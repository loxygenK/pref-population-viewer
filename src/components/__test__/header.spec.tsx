import renderer from "react-test-renderer";
import { Header } from "../header";

describe("Header component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
