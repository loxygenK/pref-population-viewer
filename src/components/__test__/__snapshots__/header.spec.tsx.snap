// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<header
  className="header"
>
  <span
    className="title"
  >
    pref-population-viewer
  </span>
</header>
`;
